packages/e2e/cypress/e2e/app/dates.cy.ts

```diff
@@ -126,144 +159,169 @@ describe('Date tests', () => {
         cy.visit(
-            `/projects/${SEED_PROJECT.project_uuid}/tables/orders${exploreStateUrlParams}`,
+            `/projects/${SEED_PROJECT.project_uuid}/tables/events${exploreStateUrlParams}`,
         );
+        cy.contains('SQL');
+        cy.findAllByText('Loading chart').should('have.length', 0);
 
+        cy.contains('1 active filter');
         cy.findByTestId('Filters-card-expand').click();
-        cy.findAllByText('Loading chart').should('have.length', 0);
-        cy.findByDisplayValue('2022-07-11, 14:23:11:000');
-        cy.get('svg g text').contains('2017-02-11, 03:00:00:000 (+00:00)');
-        cy.get('tbody td').contains('2017-02-11, 03:00:00:000 (+00:00)');
+        cy.contains('11 Aug 2020 22:58:00'); // Filter in UTC
+
+        cy.contains(`2020-08-11, 22:58:00:000 (+00:00)`); // Data in results, this comes from the server, so depends on the server timezone
+
+        // Time sensitive fields in localtime
+        const timezone = Cypress.env('TZ');
+        switch (timezone) {
+            case 'UTC':
+                cy.contains('2020-08-11 22:58:00'); // Filter in localtime //Timezone sensitive
+                break;
+            case 'Europe/Madrid':
+                cy.contains('2020-08-12 00:58:00'); // Filter in localtime //Timezone sensitive
+                break;
+            case 'America/New_York':
+                cy.contains('2020-08-11 18:58:00'); // Filter in localtime //Timezone sensitive
+                break;
+            case 'Asia/Tokyo':
+                cy.contains('2020-08-12 07:58:00'); // Filter in localtime //Timezone sensitive
+                break;
+            default:
+                throw new Error(
+                    `Invalid timezone (${timezone}) to test, please add CYPRESS_TZ to your env with a valid timezone`,
+                );
+        }
+
         cy.findByTestId('SQL-card-expand').click();
-        cy.get('code')
-            .invoke('text')
-            .should(
-                'include',
-                '("customers".created) < (\'2022-07-11 14:23:11\')',
-            );
+        cy.contains(`("events".timestamp_tz) = ('2020-08-11 22:58:00')`); // SQL
```

Comment:
This is also the same across all timezones

Reply:
Weird name though. If it's called timestamp_tz it should probably have tz information

Reply:
timestamp_tz does have timezone information, but I don't think we use it in the sql, (it is probably stripped)